### PR TITLE
Release notes and highlights for 2.6.0 release.

### PR DESCRIPTION
**Pull request by naemono**
Adding release notes and highlights for the upcoming 2.6.0 release.

### CODE REVIEW

File: docs/release-notes/highlights-2.6.0.asciidoc [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+[id="{p}-260-new-and-notable"]
+=== New and notable
+
+New and notable changes in version 2.6.0 of {n}. Check <<release-notes-2.6.0>> for the full list of changes.
+
+[float]
+[id="{p}-260-autoscaling-crd"]
+==== Elastic Stack Configuration Policy: Consistently configure multiple Elasticsearch clusters
+
+A new Custom Resource `StackConfigPolicy` was added to allow one to declaratively configure multiple Elasticsearch clusters. Elastic Stack configuration policies allow you to configure the following settings:
```
Comment (pebrc) [inline docs/release-notes/highlights-2.6.0.asciidoc:14]:
```suggestion
A new custom resource called `StackConfigPolicy` was added to allow users to declaratively configure multiple Elasticsearch clusters. Elastic Stack configuration policies allow you to configure the following settings:
```

File: docs/release-notes/2.6.0.asciidoc [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+* New CRD StackConfigPolicy to declaratively configure multiple Elasticsearch clusters. {pull}6148[#6148], {pull}6195[#6195]
+* ECK resources Helm chart - Beats. {pull}5899[#5899] (issue: {issue}5505[#5505])
+
+[[enhancement-2.6.0]]
+[float]
+=== Enhancements
+
+* Expose Kubernetes client QPS as a flag. {pull}6157[#6157]
+* Extend existing reattach-pv tool to allow using existing PVs to create newly named cluster. {pull}6118[#6118]
+* Add container-suffix operator flag to allow user to specify a container suffix to be applied across all Elastic stack container images. {pull}6086[#6086] (issue: {issue}6064[#6064])
```
Comment (pebrc) [inline docs/release-notes/2.6.0.asciidoc:22]:
```suggestion
* Add container-suffix operator flag to allow users to specify a container suffix to be applied across all Elastic stack container images. {pull}6086[#6086] (issue: {issue}6064[#6064])
```

File: docs/release-notes/2.6.0.asciidoc [hunk 3]
```diff
@@ -0,0 +1,68 @@
...
+* Improve user password hash comparison performance by utilizing an LRU cache. {pull}6080[#6080] (issue: {issue}6076[#6076])
+* Add default securityContext to the manager container in Operator Helm Chart. {pull}6047[#6047]
+* Allow Fleet Server to be run without TLS. {pull}6020[#6020] (issue: {issue}6000[#6000])
+
+[[bug-2.6.0]]
+[float]
+=== Bug fixes
+
+* Fix potential panic in Elasticsearch client equal function. {pull}6128[#6128]
+* Increment ECK-stack version to support addition of Agent/Fleet Server. {pull}6179[#6179]
```
Comment (pebrc) [inline docs/release-notes/2.6.0.asciidoc:33]:
```suggestion
* Increment ECK-stack Helm chart version to support addition of Agent/Fleet Server. {pull}6179[#6179]
```

File: docs/release-notes/2.6.0.asciidoc [hunk 4]
```diff
@@ -0,0 +1,68 @@
...
+
+[[docs-2.6.0]]
+[float]
+=== Documentation improvements
+
+* Add experimental label to the StackConfigPolicy doc. {pull}6247[#6247]
+* Document Elastic Stack configuration policies. {pull}6215[#6215]
+* Update eck-diagnostics documentation for filters. {pull}6191[#6191]
+* Add additional helm documentation for Fleet Server, and Agent. {pull}6154[#6154]
```
Comment (pebrc) [inline docs/release-notes/2.6.0.asciidoc:42]:
```suggestion
* Add additional Helm documentation for Fleet Server, and Agent. {pull}6154[#6154]
```

File: docs/release-notes/2.6.0.asciidoc [hunk 5]
```diff
@@ -0,0 +1,68 @@
...
+* Update the list of Kibana keys managed by the operator. {pull}6119[#6119]
+* Add limitation on minikube without CNI. {pull}6075[#6075]
```
Comment (pebrc) [inline docs/release-notes/2.6.0.asciidoc:44]:
```suggestion
* Document limitation on Minikube without CNI. {pull}6075[#6075]
```

File: docs/release-notes/highlights-2.6.0.asciidoc [hunk 6]
```diff
@@ -0,0 +1,31 @@
...
+== 2.6.0 release highlights
+
+[float]
+[id="{p}-260-new-and-notable"]
+=== New and notable
+
+New and notable changes in version 2.6.0 of {n}. Check <<release-notes-2.6.0>> for the full list of changes.
+
+[float]
+[id="{p}-260-autoscaling-crd"]
```
Comment (pebrc) [inline docs/release-notes/highlights-2.6.0.asciidoc:11]:
```suggestion
[id="{p}-260-stack-config-crd"]
```

**Review by thbkrkr — APPROVED** [review]
LGTM 🚀

File: docs/release-notes/highlights-2.6.0.asciidoc [hunk 7]
```diff
@@ -0,0 +1,31 @@
...
+[id="{p}-260-new-and-notable"]
+=== New and notable
+
+New and notable changes in version 2.6.0 of {n}. Check <<release-notes-2.6.0>> for the full list of changes.
+
+[float]
+[id="{p}-260-stack-config-crd"]
+==== Elastic Stack Configuration Policy: Consistently configure multiple Elasticsearch clusters
+
+A new custom resource called `StackConfigPolicy` was added to allow users to declaratively configure multiple Elasticsearch clusters. Elastic Stack configuration policies allow you to configure the following settings:
```
[the remark this answers is not in the document]
Comment (naemono) [reply docs/release-notes/highlights-2.6.0.asciidoc]: @thbkrkr I accepted this with a slight variation in the wording.  LMK what you think.

[the remark this answers is not in the document]
Comment (thbkrkr) [reply docs/release-notes/highlights-2.6.0.asciidoc]: LGTM

File: docs/release-notes/highlights-2.6.0.asciidoc [hunk 8]
```diff
@@ -0,0 +1,36 @@
...
+[float]
+[id="{p}-260-agent-fleet-helm-chart"]
+==== Helm chart for Elastic Beats
+
+We are adding a new Helm chart for workloads managed by the ECK operator. Elastic Beats are complementing the existing charts for Elasticsearch, Kibana, Agent and Fleet Agent. They can be used individually or as part of our existing Elastic Stack Helm chart. Get started by reading the  <<{p}-stack-helm-chart, docs>>.
+
+[float]
+[id="{p}-260-known-issues"]
+=== Known issues
+- If a license for an Elasticsearch cluster expires before being replaced with an updated license the ECK Operator cannot automatically update the license. The underlying link:https://github.com/elastic/cloud-on-k8s/issues/6274[issue] is resolved in the following link:https://github.com/elastic/cloud-on-k8s/pull/6278[pull request] and will be released in an upcoming release of the Operator.
```
Comment (pebrc) [inline docs/release-notes/highlights-2.6.0.asciidoc:36]:
```suggestion
- If a license for an Elasticsearch cluster expires before being replaced with an updated license the ECK operator cannot automatically update the license. See the underlying link:https://github.com/elastic/cloud-on-k8s/issues/6274[issue] for a description of workarounds. A fix will be released in the next release of the operator.
```

**Review by pebrc — APPROVED** [review]
LGTM

**Comment by naemono** [comment]
## 💚 All backports created successfully

| Status | Branch | Result |
|:------:|:------:|:------|
|✅|2.6|[](https://github.com/elastic/cloud-on-k8s/pull/6288)|

### Questions ?
Please refer to the [Backport tool documentation](https://github.com/sqren/backport)

**Pull request merged**